src/NuGetForUnity/Editor/Helper/NugetPackageTextureHelper.cs

```diff
@@ -116,7 +116,15 @@ private static string GetHash(string s)
 
             var md5 = new MD5CryptoServiceProvider();
             var data = md5.ComputeHash(Encoding.Default.GetBytes(s));
-            return Convert.ToBase64String(data);
+
+            // Convert byte array to hexadecimal string
+            var sb = new StringBuilder();
+            for (int i = 0; i < data.Length; i++)
+            {
+                sb.Append(data[i].ToString("x2"));
+            }
+
+            return sb.ToString();
```

Comment:
```suggestion
            return Convert.ToBase64String(data).Replace('+', '-').Replace('/', '_').Replace("=", null);
```
It would basically be Base64Url encoding. So only `[A-Za-z]`, `'-'` and `'_'`. Should be supported by all operating systems. And we will have a single line 😄